R-package/R/lgb.Dataset.R

```diff
@@ -710,7 +710,9 @@ Dataset <- R6::R6Class(
 #' @title Construct \code{lgb.Dataset} object
 #' @description Construct \code{lgb.Dataset} object from dense matrix, sparse matrix
 #'              or local file (that was created previously by saving an \code{lgb.Dataset}).
-#' @param data a \code{matrix} object, a \code{dgCMatrix} object or a character representing a filename
+#' @param data a \code{matrix} object, a \code{dgCMatrix} object,
+#'             a character representing a path to a text file (CSV, TSV, or LibSVM)
+#'             or a LightGBM Dataset binary file
```

Comment:
Let's make identical descriptions really identical without paraphrasing. Users might spend their time trying to find a difference where there is actually no any difference. Also, it will help us to not miss all occurrences of identical parameters during possible future updates.
```suggestion
#'             a character representing a path to a text file (CSV, TSV, or LibSVM),
#'             or a character representing a path to a binary \code{Dataset} file
```

Reply:
oh sure, seems fine to me!

I'll do this one manually instead of applying in the browser, since the corresponding `lgb.Dataset.Rd` will also have to be regenerated.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/4545/commits/de189a677a0eb5a69536dfe409744871b95055be, thanks as always for being thorough! You're right, that will make it easier to catch all such phrases in the future.